.github/workflows/ci.yml

```diff
@@ -57,6 +63,10 @@ jobs:
           submodules: recursive
           # Needed for script/package.sh to work
           fetch-depth: 0
+      - name: Use go ${{ matrix.go }}
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go }}
```

Comment:
I needed to enforce a newer version of go (by default it used 1.15) otherwise go wouldn't be able to link git-lfs for arm64.